kalitectl.py

```diff
@@ -400,7 +400,7 @@ def start_watchify():
     sys.stdout.write('Starting watchify')
 
     watchify_process = subprocess.Popen(
-        ['node compile_javascript.js --debug --watch --staticfiles'],
+        args='node compile_javascript.js --debug --watch --staticfiles',
```

Comment:
list to string. But it's still perfectly valid python. It seems [any argument can be referenced by name](https://docs.python.org/2/tutorial/controlflow.html#keyword-arguments), whether the argument is required or not (require==does not have a default value).